homework/hackyourtemperature/server.js

```diff
@@ -1,19 +1,24 @@
 import express from "express";
 
+//Start app
 const app = express();
 
+//Middleware to have requests as JSON
 app.use("/weather", express.json());
 
+//Default path
 app.get("/", (req, res) => {
+  res.type("html");
   res.status(200).send("<p>Hello from backend to frontend!</p>");
 });
 
+//Post city name in the body
 app.post("/weather", (req, res) => {
   if (req.body?.cityName) {
     const cityName = req.body.cityName;
-    res.status(200).send(cityName);
+    res.status(200).json({ cityName });
   } else {
-    res.status(404).send("Could not find city in request");
+    res.status(400).send("Could not find city in request");
```

Comment:
Almost there! 
In general it is better if all responses on an API are consistent on their data type. 
If I am working with and API that gives me data in JSON format, I also expect the errors to follow the same pattern. 
```suggestion
    res.status(400).json({message: "Could not find city in request"});
```